recipes/lua-5.1/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "5.1.5" %}
+
+package:
+  name: lua
+  version: {{ version }}
+
+source:
+  fn: lua-5.1.5.tar.gz
```

Comment:
Done

On Thu, May 12, 2016 at 2:01 PM Mike Sarahan <notifications@github.com>
wrote:

> In recipes/lua-5.1/meta.yaml
> <https://github.com/conda-forge/staged-recipes/pull/585#discussion_r63077652>
> :
>
> > @@ -0,0 +1,40 @@
> > +{% set version = "5.1.5" %}
> > +
> > +package:
> > +  name: lua
> > +  version: {{ version }}
> > +
> > +source:
> > +  fn: lua-5.1.5.tar.gz
>
> Sorry I wasn't clear, you can also use the template variable here, and in
> the URL.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/conda-forge/staged-recipes/pull/585/files/b7f441fe503865a9a3b3aca3d0eb72005fbeaf83#r63077652>
>